libkbfs/folder_branch_ops.go

```diff
@@ -19,6 +18,7 @@ import (
 	"github.com/keybase/kbfs/kbfscrypto"
 	"github.com/keybase/kbfs/kbfssync"
 	"github.com/keybase/kbfs/tlf"
+	"github.com/pkg/errors"
```

Comment:
if you're doing this, can you also replace `fmt.Errorf` with `errors.Errorf` in this file?